src/CommunityVoices/App/Website/Presentation/Module/Form/Quote.xslt

```diff
@@ -67,7 +82,8 @@
                                 <xsl:when test="domain/form != ''">
                                     <xsl:value-of select="domain/form/text"/>
                                 </xsl:when>
-                                <xsl:when test="domain/quote != ''">
+                                <xsl:when test="
```

Comment:
This may have been a mistake, don't know why I would've changed this

Reply:
This would be the bug that we discovered with John where the text was not being filled in.